packages/core/src/Chip/ChoiceChip/ChoiceChip.stories.tsx

```diff
@@ -203,6 +203,34 @@ export const CustomChip = () => (
   </Box>
 )
 
+// With Box Shadow
+export const WithBoxShadows = () => (
+  <Box>
+    {[
+      { boxShadowSize: 'sm', heading: 'Small Shadow', label: 'Enabled' },
+      { boxShadowSize: 'md', heading: 'Medium Shadow', label: 'Active', selected: true },
+      { boxShadowSize: 'lg', heading: 'Large Shadow', label: 'Disabled', disabled: true },
+      {
+        boxShadowSize: 'xl',
+        heading: 'XLarge Shadow',
+        label: 'Active and Disabled',
+        selected: true,
+        disabled: true,
+      },
+    ].map((props) => {
```

Comment:
Could you export this array from here so it can be reused in the three places you have it now?

https://github.com/priceline/design-system/blob/main/packages/core/src/storybook/args/index.ts